### PR TITLE
Move assessment merge requirement

### PR DESCRIPTION
**Pull request by lumjjb**
Signed-off-by: Brandon Lum <lumjjb@gmail.com>

### CODE REVIEW

**Comment by lumjjb** [comment]
@ultrasaurus

File: assessments/guide/security-reviewer.md [hunk 1]
```diff
@@ -36,13 +36,6 @@ The level of effort for the reviewers is expected to be 10 hours per review. Cor
 * attending project presentation with Q&A
 * discussion via Slack or additional live Q&A sessions
 
-## Expectations
-
-GitHub issues are assigned to security reviewers. Security reviewers are
-expected to conduct an in-depth review described in the [Security Assessment
-Guide](./). Security reviewers should seek the approval of the other
-participating security reviewers and at least 1 co-chair before merging.
-
```
Comment (JustinCappos) [inline assessments/guide/security-reviewer.md:45]: Is this effectively being moved to the items above?  Is this the rationale for this removal?

Comment (lumjjb) [reply assessments/guide/security-reviewer.md:45]: I remember the motivation for this was that this is part of the review process, but it seems like only the last line here is relevant. So I would be in favor of keeping this (minor the last sentence).
```
Security reviewers should seek the approval of the other
participating security reviewers and at least 1 co-chair before merging.```

**Comment by lumjjb** [comment]
Updated to reflect feedback

**Pull request merged**